Tests/run_tests.py

```diff
@@ -257,7 +258,13 @@ def main(argv):
             # This is a bit of a hack...
             import requires_internet
             requires_internet.check.available = False
-            # The check() function should now report internet not available
+            # Monkey patch for urlopen()
+            import Bio._py3k
+
+            def dummy_urlopen(url):
+                raise RuntimeError("internet not available")
```

Comment:
Can you make that error message something like: ``Internal test suite error, attempting to use internet despite --offline setting.``